docs/release_notes.rst

```diff
@@ -1,6 +1,40 @@
 Release Notes
 =============
 
+v3.0.0
+-----
+
+:date: 2017-06-09
+
+This is a major release with breaking changes.
+
+``MapAttribute`` now allows pythonic access when recursively defined.
+If you were not using the ``attr_name=`` kwarg then you should have no problems upgrading.
+Previously defined non subclasesd ``MapAttributes`` (raw ``MapAttributes``) that were members of a subclassed ``MapAttribute`` (typed ``MapAttributes``) would have to be accessed like a dictionary.
+Now object access is possible and recommended. See [here](https://github.com/pynamodb/PynamoDB/blob/master/pynamodb/tests/test_attributes.py#L671) for a test example.
+Object notation with the ``attr_name`` also known as the DynamoDB name will now throw an ``AttributeError``.
```

Comment:
Access via the ``attr_name``, also known as the DynamoDB name, will now throw an ``AttributeError``